src/libraries/System.Runtime.InteropServices/gen/Microsoft.Interop.SourceGeneration/GeneratedStatements.cs

```diff
@@ -18,6 +18,7 @@ public struct GeneratedStatements
         public ImmutableArray<StatementSyntax> Setup { get; init; }
         public ImmutableArray<StatementSyntax> Marshal { get; init; }
         public ImmutableArray<FixedStatementSyntax> Pin { get; init; }
+        public ImmutableArray<StatementSyntax> PinnedMarshal { get; init; }
         public StatementSyntax InvokeStatement { get; init; }
         public ImmutableArray<StatementSyntax> Unmarshal { get; init; }
         public ImmutableArray<StatementSyntax> KeepAlive { get; init; }
```

Comment:
We should update this `NotifyForSuccessfulInvoke`. Also where is `UnmarshalCapture`?

Reply:
I decided to roll UnmarshalCapture into the Unmarshal statement list as UnmarshalCapture always comes directly before Unmarshal. We can have them separate if we so desire.